decentraland/unity-explorer
# feat: loading screen polishing

Introduce visual improvements on the loading screens:
- background blur
- transition animations
- layout
- randomization fixes
- tip rotation duration

Also introduces the loading screens when teleporting to any parcel, either from the debug panel or the minimap.

# How to test

1. Start the initial flow as expected
2. Change realm from the debug panel
3. Teleport to different spawn points or coordinates
4. Rotate tips
5. Check that the loading screen is always displayed as expected
6. Open the minimap and click to jumpin to any parcel
